TelCo.ColorCoder/GetColorFromPairNumber.cs

```diff
@@ -0,0 +1,41 @@
+using System;
+using System.Diagnostics;
+using System.Drawing;
+
+namespace TelCo.ColorCoder
```

Comment:
Naming convetion for files and classes:
A general guideline is have the name of file and class it contains to be same. 
Have single class in a single file.
File name with function name it contains is not right.

Class name should be a noun and function names should be verb.
and hence File name would be noun too.